versioned_docs/version-3.10/operator-guide/cd/argocd-and-kuberocketci-statuses.md

```diff
@@ -0,0 +1,244 @@
+---
+title: "Argo CD & KubeRocketCI: Understanding Statuses"
```

Comment:
let's remove ArogCD from the title. and we need to make it more clear "Understanding statuses??? what"